ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+set -ex
+
+#sudo apt-get install -y rocblas rccl-dev rccl-rdma-sharp-plugins
+sudo apt install libstdc++-12-dev
+sudo apt remove -y rccl
+cd ~
+git clone https://github.com/rocm/rccl
+cd rccl
+mkdir build
+cd build
+CXX=/opt/rocm/bin/hipcc cmake -DCMAKE_PREFIX_PATH=/opt/rocm/ -DCMAKE_INSTALL_PREFIX=/opt/rccl ..
+make -j 32
+sudo make install
+cd ~
+
+
+sudo sysctl kernel.numa_balancing=0
+echo "kernel.numa_balancing=0" | sudo tee -a /etc/sysctl.conf
+
+
+git clone https://github.com/ROCmSoftwarePlatform/rccl-tests
+cd rccl-tests
+
+source /opt/hpcx-v2.16-gcc-mlnx_ofed-ubuntu22.04-cuda12-gdrcopy2-nccl2.18-x86_64/hpcx-init.sh
```

Comment:
Is it possible to use the hpcx environment module at this point, and use some of the environment variables once the module instead of hardcoding HPCX var?

Reply:
Especially since that will change with a new version of HPCX

Reply:
How about I source /opt/hpcx*/hpcx-init.sh ?